ruby_on_rails/app_initialisation.md

```diff
@@ -29,8 +34,8 @@ If you do not need a DB you may rethink the fact that you may not need Rails at
 
 * Run `bundle exec rails db:migrate` to generate an empty `schema.rb` file.
 
-* Then check your default Rails setup by running `rails s` and visiting `localhost:3000`.
-  You should be on Rails now, yay!
+* Check your default Rails setup by running either `bin/dev` or `rails s`, depending on whether you are using a node-based bundler with the foreman Procfile-runner, or if you are using importmaps and then visit `localhost:3000`.
```

Comment:
At this point of the story, `bin/rails s` will work anyway since there are no assets in the homepage, right?

Reply:
Yes I just tried it, thanks